CHANGELOG.md

```diff
@@ -18,8 +18,8 @@
 - `windowsperfcountersreceiver`: Move code into separate package for use in other windowsperfcounter receivers (#9108) 
 - `datadogexporter`: Add `host_metadata` configuration section to configure host metadata export (#9100)
 - `cmd/mdatagen`: Update documentation generated for attributes to list enumerated values and show the "value" that will be visible on metrics when it is different from the attribute key in metadata.yaml (#8983)
-
 - `routingprocessor`: add option to drop resource attribute used for routing (#8990)
+- `cumulativetodeltaprocessor`: add new include/exclude configuration options with regex support (#8952)
```

Comment:
We should also have a deprecation note for the `metrics` option

Reply:
Want me to add that to this line or make a second entry for the processor in "Deprecations"?

Reply:
I think adding it to 'Deprecations' is cleaner, but up to you :)